src/components/TutorialCarouselDialog/components/buttonsWrapper.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Typography, withStyles } from 'material-ui';
+import Dialog, { DialogActions, DialogContent, DialogTitle } from 'material-ui/Dialog';
+import cx from 'classnames';
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import _ from 'lodash';
+
+import styles from '../styles';
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
+export default class TutorialButtonsWrapper extends Component {
```

Comment:
everything looks good except I still don't know where we're using this. I don't see it imported in any files (unless I'm just missing something) :)